core/src/main/scala/cromwell/core/callcaching/package.scala

```diff
@@ -0,0 +1,67 @@
+package cromwell.core
+
+package object callcaching {
+
```

Comment:
For instance, all of the `HashValue` stuff could go in its own file. The `CallCachingMode` ADT could go in its own file. The `ReadWriteMode` ADT could go in its own file. etc